linta-android/build.gradle.kts

```diff
@@ -38,3 +41,38 @@ tasks.withType<Jar> {
         attributes["Lint-Registry-v2"] = "com.swvl.lint.LintIssueRegistry"
     }
 }
+
+mavenPublishing {
```

Comment:
let's add the logo here like the old way to give it a try in the next release, wdyt?